docs/examples/sign-and-verify-messages.md

```diff
@@ -0,0 +1,14 @@
+## Sign and verify messages
+
+DashJS exports the Message constructor `new DashJS.Message`.   
+
+You can refer to its documentation : https://github.com/dashevo/dashcore-message/blob/master/README.md
+
+```js
+const pk = new DashJS.PrivateKey();
```

Comment:
Don't it should be `Dash.Core.PrivateKey();`?

Reply:
Good eyes, I have fixed this along with other mentions in #39